tests/fuzz/s2n_hybrid_ecdhe_sike_fuzz_test.c

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include <pq-crypto/sike/sike_p503_kem.h>
+#include "crypto/s2n_crypto.h"
+#include "crypto/s2n_drbg.h"
+#include "crypto/s2n_hash.h"
+#include "crypto/s2n_openssl.h"
+
+#include "stuffer/s2n_stuffer.h"
+
+#include "tests/s2n_test.h"
+
+#include "tests/testlib/s2n_testlib.h"
+
+#include "tests/unit/s2n_nist_kats.h"
+
+#include "tls/s2n_kex.h"
+#include "tls/s2n_kem.h"
+#include "tls/s2n_tls.h"
+
+#include "utils/s2n_random.h"
+#include "utils/s2n_safety.h"
+#include "utils/s2n_safety.h"
+#include "tls/s2n_cipher_suites.h"
+
+#include "error/s2n_errno.h"
+
+
+static struct s2n_kem_keypair server_kem_keys = {.negotiated_kem = &s2n_sike_supported_params[0]};
+
+static int setup_connection(struct s2n_connection *server_conn)
+{
+    server_conn->actual_protocol_version = S2N_TLS12;
+    server_conn->secure.server_ecc_params.negotiated_curve = &s2n_ecc_supported_curves[0];
+    server_conn->secure.s2n_kem_keys.negotiated_kem = &s2n_sike_supported_params[0];
+    server_conn->secure.cipher_suite = &s2n_ecdhe_sike_rsa_with_aes_256_gcm_sha384;
+    server_conn->secure.conn_hash_alg = S2N_HASH_SHA384;
+
+    GUARD(s2n_dup(&server_kem_keys.private_key, &server_conn->secure.s2n_kem_keys.private_key));
+    /* s2n_ecdhe_client_key_recv will free the server's point so it needs to be recreated every run */
```

Comment:
A method level comment about the expected connection state you need to setup might be more helpful. I was unclear why you called out copying the server_ecc_params but didn't call out copying the private_key.

Reply:
Updated the comment.